Templates/Ros2FleetRobotTemplate/Template/project.json

```diff
@@ -0,0 +1,27 @@
+{
+    "project_name": "${Name}",
```

Comment:
question: This might be the issue, should this name be changed? 🤔

Reply:
Same as above, this is the name of the project, not the template.